app/helpers/search_bar_helper.rb

```diff
@@ -25,4 +25,24 @@ def mount_search_bar(
       bookmarks: bookmarks
     }.to_json)
   end
+
+  def get_search_props(
```

Comment:
we might need to introduce a search controller at some stage, as if we don't have a controller per page for react, we won't have an endpoint, having said that, we might just add it as part of our API instead..?